subprojects/core-api/src/main/java/org/gradle/StartParameter.java

```diff
@@ -698,6 +699,26 @@ public void setBuildCacheEnabled(boolean buildCacheEnabled) {
         this.buildCacheEnabled = buildCacheEnabled;
     }
 
+    /**
+     * Whether build cache debug logging is enabled.
+     *
+     * @since 4.6
+     */
+    @Incubating
+    public boolean isBuildCacheDebugLogging() {
+        return buildCacheDebugLogging;
+    }
+
+    /**
+     * Whether build cache debug logging is enabled.
+     *
+     * @since 4.6
+     */
+    @Incubating
+    public void setBuildCacheDebugLogging(boolean buildCacheDebugLogging) {
+        this.buildCacheDebugLogging = buildCacheDebugLogging;
+    }
+
```

Comment:
Can we put this into `StartParameterInternal` instead?

Reply:
I put it on the public API, since the property also is public. So it is possible to enable this property programmatically.
What would be your reason to make this property internal?